Source/FieldSolver/SpectralSolver/SpectralFieldDataRZ.cpp

```diff
@@ -409,19 +419,26 @@ SpectralFieldDataRZ::BackwardTransform (amrex::MultiFab& field_mf_r, int const f
     amrex::MultiFab tempHTransformedSplit_m(tempHTransformed.boxArray(), tempHTransformed.DistributionMap(), 2*n_rz_azimuthal_modes, 0);
 
     // Create copies of the input multifabs. The copies will include the imaginary part of mode 0.
-    amrex::MultiFab field_mf_r_copy(field_mf_r.boxArray(), field_mf_r.DistributionMap(), 2*n_rz_azimuthal_modes, field_mf_r.nGrowVect());
-    amrex::MultiFab field_mf_t_copy(field_mf_t.boxArray(), field_mf_t.DistributionMap(), 2*n_rz_azimuthal_modes, field_mf_t.nGrowVect());
+    amrex::IntVect guards;
```

Comment:
I had missed this in my most recent commit. As in the ForwardTransform, this bit of code can be cleaned up by declaring the Multifabs like this:
    amrex::MultiFab field_mf_r_copy(tempHTransformed.boxArray(), field_mf_r.DistributionMap(), 2*n_rz_azimuthal_modes, 0);
    amrex::MultiFab field_mf_t_copy(tempHTransformed.boxArray(), field_mf_t.DistributionMap(), 2*n_rz_azimuthal_modes, 0);

The guards variable can them be deleted. This also allows removing the `m_periodic_single_box` attribute from this class since it's not used anywhere else. Then it also can be removed from SpectralSolverRZ.

Reply:
@dpgrote @RemiLehe I addressed these comments in https://github.com/ECP-WarpX/WarpX/pull/1301/commits/a01283ff99ab8c7cef5f6adbaf2aa853ad0236a9. In addition, I also removed the argument `pml` from the constructor of `SpectralSolverRZ`, because this argument was not used and it was causing a compiler warning.